tom/dependencies.py

```diff
@@ -1,6 +1,7 @@
 import re
 import json
 import requests
+import collections
```

Comment:
I don't see any use of `collections` in the new lines below. Was it missing or can the import be dropped from this commit?

Reply:
wow, these lines were supposed to be in another commit... Was I drunk when submitting this PR?